assembl/static2/js/app/components/debate/common/ideas.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { Translate } from 'react-redux-i18n';
+import { Grid } from 'react-bootstrap';
+import '../../../../../css/components/ideas.scss';
```

Comment:
Please remove this import, it doesn't take into account the theme. And you already added it in _assembl_base_styles.scss anyway.